swagger-templates/csharp/ApiClient.mustache

```diff
@@ -145,6 +154,10 @@ namespace {{packageName}}.Client
                 {
                     request.AddParameter(contentType, postBody, ParameterType.RequestBody);
                 }
+            } 
+            else 
+            {
```

Comment:
Is this for the empty request body case? Would it be necessary to add a comment explaining that our API explicitly requires an empty JSON body for requests that don't take parameters?